docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -35,7 +30,10 @@ exemplars can be automatically added, providing additional value to these metric
 
 ## How to run
 
-To enable span metrics in Tempo/GET, enable the metrics generator and add an overrides section which enables the `span-metrics` generator. See [here for configuration details]({{< relref "../configuration#metrics-generator" >}}).
+To enable span metrics in Tempo or Grafana Enterprise Traces, enable the metrics generator and add an overrides section which enables the `span-metrics` generator.
```

Comment:
I know this wording already existed, but I'm wondering why we are using `generator` here? We usually refer to them as processors.
```suggestion
To enable span metrics in Tempo or Grafana Enterprise Traces, enable the metrics generator and add an overrides section which enables the `span-metrics` processor.
```

Reply:
lYou know, we probably  should do a general review of the docs for terminology so we're consistent.